src/geometry/manifold/algebra/lie_group.lean

```diff
@@ -24,13 +24,6 @@ groups here are not necessarily finite dimensional.
                         corners `I`.
 * `lie_add_group_morphism I I' G G'`  : morphism of addittive Lie groups
 * `lie_group_morphism I I' G G'`      : morphism of Lie groups
-* `lie_add_group_core I G`            : allows to define a Lie additive group without first proving
-                                        it is a topological additive group.
-* `lie_group_core I G`                : allows to define a Lie group without first proving
-                                        it is a topological group.
-
-* `reals_lie_group`                   : real numbers are a Lie group
```

Comment:
I gather you're removing this because at some point `reals_lie_group` was generalized to `normed_space_lie_group`, could you instead add a line
```suggestion
* `normed_space_lie_group`                   : normed space over a nondegenerate normed field is a Lie group
```

Reply:
Yes, I searched for "reals_lie_group" and found nothing. I'll add `normed_space_lie_group` to the docs tomorrow.